src/components/Navbar/index.jsx

```diff
@@ -1,30 +1,46 @@
-import { AppBar, Toolbar, Typography, Grid, Box, Button, IconButton, Menu, MenuList, MenuItem } from "@mui/material";
-import React, { useRef, useState } from "react";
+import { AppBar, Toolbar, Typography, Grid, Box, Button, IconButton, Menu, MenuList, Select, MenuItem, Paper } from "@mui/material";
+import React, { useRef, useState, useEffect } from "react";
 import logo from "/images/Logo.png";
 import MenuIcon from "@mui/icons-material/Menu";
-import ArrowDropDown from "@mui/icons-material/ArrowDropDown";
 import { Link } from "react-router-dom";
-import PropTypes from "prop-types";
+import { getRegion } from "../../api/region";
 
-const Navbar = ({ region }) => {
+const Navbar = () => {
+  const [regions, setRegions] = useState([]);
+  const [selectedRegion, setSelectedRegion] = useState("");
   const [navStatus, setNavStatus] = useState(false);
   const iconButtonRef = useRef();
+
```

Comment:
since we it's not recommended to use `async await` inside useEffect, I created `fetch` function outside, then invoke it inside useEffect to get the region data.